README.md

```diff
@@ -12,13 +12,16 @@ This is the server component of the AeroGear metrics service. It is a RESTful AP
 
 ## Clone and Install Dependencies
 
-1. Clone this repository
+1. Clone this repository to `$GOPATH/src/github.com/aerogear/aerogear-metrics-api`
+```sh
+mkdir -p $GOPATH/src/github.com/aerogear
+cd $GOPATH/src/github.com/aerogear
+gir clone git@github.com:aerogear/aerogear-metrics-api.git
 ```
-go get github.com/aerogear/aerogear-metrics-api
-```
-1. Run the following command to build binary
+
+2. Run the following command install dependencies
```

Comment:
Step not needed.

Reply:
There should still be a very explicit step in the Readme on Installing Deps even if the step is to run something with `make`